### PR TITLE
added deleting the container from myraid on container stop

### PR DESCRIPTION
**Pull request by ashleyschuett**

### CODE REVIEW

**Comment by normanjoyner** [comment]
@ashleyschuett please sync

File: engines/docker-engine.js [hunk 1]
```diff
@@ -321,7 +322,16 @@ class DockerEngine extends Engine {
                     return this.log('error', `Failed to stop container ${options.container_id}: ${err}`);
                 }
 
-                this.log('info', `Sucessfully stopped container ${options.container_id}`);
+                Utils.deleteContainerMyriadState(this.core, {
+                    application_name: options.application,
```
Comment (normanjoyner) [inline engines/docker-engine.js]: should this be `options.application_name`?

Comment (ashleyschuett) [inline engines/docker-engine.js]: It's under `options.application`, if you look at `lib/follower.js` line 167.

Comment (normanjoyner) [inline engines/docker-engine.js]: nope, it seems like `options.application` should be correct. just traced it back through core

**Pull request merged**